lib/segment/src/fixtures/payload_context_fixture.rs

```diff
@@ -0,0 +1,204 @@
+use crate::entry::entry_point::OperationResult;
+use crate::fixtures::payload_fixtures::{
+    generate_diverse_payload, FLT_KEY, GEO_KEY, INT_KEY, STR_KEY,
+};
+use crate::id_tracker::points_iterator::PointsIterator;
+use crate::id_tracker::IdTracker;
+use crate::index::plain_payload_index::PlainPayloadIndex;
+use crate::index::struct_payload_index::StructPayloadIndex;
+use crate::index::PayloadIndex;
+use crate::payload_storage::in_memory_payload_storage::{
+    InMemoryConditionChecker, InMemoryPayloadStorage,
+};
+use crate::payload_storage::PayloadStorage;
+use crate::types::{PayloadSchemaType, PointIdType, PointOffsetType, SeqNumberType};
+use atomic_refcell::AtomicRefCell;
+use rand::prelude::StdRng;
+use rand::SeedableRng;
+use std::path::Path;
+use std::sync::Arc;
+
+struct IdsIterator {
+    ids: Vec<PointOffsetType>,
+}
+
+/// Warn: Use for tests only
+///
+/// This struct mimics the interface of `PointsIterator` and `IdTracker` only for basic cases
+impl IdsIterator {
```

Comment:
I am not sure it is recognized as doc comment otherwise.
```suggestion

/// Warn: Use for tests only
///
/// This struct mimics the interface of `PointsIterator` and `IdTracker` only for basic cases
struct IdsIterator {
    ids: Vec<PointOffsetType>,
}

impl IdsIterator {
```